Bio/PDB/Dice.py

```diff
@@ -42,8 +42,8 @@ def accept_residue(self, residue):
             # skip HETATMS
             return 0
         if icode != " ":
-            warnings.warn("WARNING: Icode at %s" % residue.get_id(),
-                          RuntimeWarning)
+            warnings.warn("WARNING: Icode %s at position %s" \
+                          % (icode, resseq), RuntimeWarning)
```

Comment:
Perhaps this should be ``BiopythonWarning``? You'd need to add this at the top of the file:

``` python
from Bio import BiopythonWarning
```